### DISCUSSION TITLE
Can ezdxf parse the data in the dwg file?

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#692] [General]
Does ezdxf support reading file data in dwg format?

**Comment by mozman** [comment]
Not directly, the name of package shows its main purpose:  `ezdxf`.

The [odafc](https://ezdxf.mozman.at/docs/addons/odafc.html) add-on can utilize the [Open Design Alliance File Converter](https://www.opendesign.com/guestfiles/oda_file_converter) to convert DWG to DXF and back. You need to install the ODAFC application yourself!
